packages/dashboard/src/app/views/shared/bodyViewOptions.js

```diff
@@ -36,10 +36,14 @@ import { StandardViewContentGutter, ViewStyleBar } from '../../../components';
 import { DROPDOWN_TYPES, VIEW_STYLE } from '../../../constants';
 import useFilters from '../myStories/filters/useFilters';
 
+const FILTER_MAX_WIDTH = 350;
```

Comment:
Can this match the width of the sorting menu? In the example 'after' image, that's the widest one besides the super-wide menu.